lesson3/Lesson3HW/app/src/main/res/layout/activity_main.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Making this a LinearLayout is one way to make sure your fragment doesn't overlap with the toolbar